baton/handler.go

```diff
@@ -14,6 +15,8 @@ func Handler() *mux.Router {
 	return m
 }
 
+var i = 0
```

Comment:
This is simply something to give each user a unique identifier. No need to store put it in a database.